folding/utils/logging.py

```diff
@@ -80,5 +83,13 @@ def log_event(self, event):
     if not getattr(self, "wandb", None):
         init_wandb(self)
 
+    if getattr(self, "wandb_run_start", None):
+        if (dt.datetime.now() - self.wandb_run_start) >= dt.timedelta(days=1):
```

Comment:
I wonder if the better way to do this is just by checking the number of lines that have been printed to the terminal. but keeping track of this would be super annoying. are you sure that a day is a good metric of the upper line limit? (10_000)

Reply:
Yeah I mostly used 1 day because it was easiest to implement. If you look at this wandb run start time: https://wandb.ai/macrocosmos/folding-validators/runs/5ng6jvqg/overview and compare that to the last logged log line: https://wandb.ai/macrocosmos/folding-validators/runs/5ng6jvqg/logs you will see that the Validator succesfully logged for a bit more than 24 hours before reaching the upper limit of 100_000 lines

Reply:
i wouldn't use that validator log for metrics. they blacklisted everyone except their miners thats why logged more than usual. but 1 day usually does it if they dont have trace on